daq/host.py

```diff
@@ -324,15 +325,16 @@ def _state_transition(self, target, expected=None):
     def _build_switch_info(self) -> usi.SwitchInfo:
         switch_config = self._get_switch_config()
         model_str = switch_config['model']
-        if model_str == 'FAUX_SWITCH' or not self.target_port:
+        target_port = self.target_port or self.nonof_dev_port
```

Comment:
case in point -- this logic should be unnecessary.

Reply:
Changed.